go.mod

```diff
@@ -1,66 +1,72 @@
 module k8s.io/cloud-provider-gcp
 
 require (
-	k8s.io/api v0.19.0
-	k8s.io/apimachinery v0.19.0
-	k8s.io/apiserver v0.19.0
-	k8s.io/client-go v0.19.0
-	k8s.io/cloud-provider v0.19.0
-	k8s.io/component-base v0.19.0
-	k8s.io/kube-controller-manager v0.19.0
-	k8s.io/kubernetes v1.19.0
-	k8s.io/legacy-cloud-providers v0.19.0
+	k8s.io/api v0.20.0
+	k8s.io/apimachinery v0.20.0
+	k8s.io/apiserver v0.20.0
+	k8s.io/client-go v0.20.0
+	k8s.io/cloud-provider v0.20.0
+	k8s.io/component-base v0.20.0
+	k8s.io/kubelet v0.20.0
+	k8s.io/kubernetes v1.20.0
```

Comment:
It would be good if we could remove the k8s.io/kubernetes dependency. Do we know why we have it? We should not be relying on anything K/K private.

Reply:
Updated. (It was there because previous NodeIPAM related dependencies. Sorry)